src/summary/documentation/dimensions-card.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import { match } from "react-router";
+import { getPatientDimensions } from "./dimensions.resource";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalSectionCard from "./horizontal-section-card.component";
+import DimensionsSectionCard from "./dimensions-card-row.component";
+import { ageConvert } from "./date-helper";
+import ShowMoreCard from "./show-more-card.component";
+import style from "./dimensions-card.css";
+export default function DimensionsCard(props: DimensionsCardProps) {
+  const [dimensions, setDimensions] = React.useState([]);
+  const [showElements, setShowElements] = React.useState(3);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+    getPatientDimensions(
+      props.currentPatient.identifier[0].value,
+      abortController
+    ).then(response => {
+      const ar = response.data.entry
+        .filter(
+          el =>
+            el.resource.valueQuantity !== undefined &&
+            (el.resource.valueQuantity.unit === "cm" ||
+              el.resource.valueQuantity.unit === "kg" ||
+              el.resource.valueQuantity.unit === undefined)
+        )
+        .map(m => m.resource);
+      const res = [];
+      for (let i = 0; i < ar.length; i += 3) {
```

Comment:
Why are we incrementing i by three every time? It seems like we're depending on the `data.entry` to be in a very specific order that may not hold true.

Reply:
They are not ordered by date? This was an observation of mine and i saw that it works i don't know if it was a coincidence.

Reply:
The algorithm here just seems quite arbitrary. There is no clear reason for `i+=3` to the reader of this code.